server/dao/user.js

```diff
@@ -13,6 +13,17 @@ class UserDao {
     return user;
   };
 
+  getUserViaEmail = async (email) => {
+    
+    const {
+      rows,
+    } = await this.dbPool.query('SELECT * FROM rentr_user WHERE email = $1;', [
+      email,
+    ]);
+    const user = rows && rows.length >= 1 ? rows[0] : null;
```

Comment:
This could potentially be reduced to
`return rows && rows.length >= 1 ? rows[0] : null;`

Reply:
Either looks fine to me. Ronnie's way would be more explicit for readers, while Nathan's way is more simple.